api/MediaRecorder.json

```
@@ -131,7 +131,7 @@
                 "version_added": "14"
               },
               "safari_ios": {
-                "version_added": "14"
+                "version_added": "14.3"
```

```suggestion
                "version_added": "14"
```

This is the only remaining occurrence of "14.3", all others changed to "14", so I presume the intention in https://github.com/mdn/browser-compat-data/pull/10129#discussion_r622381020 was to treat "14.3" as "14" for `safari_os`.

---

Addressed in a086511.